Add/fix XML doc comments for RepositoryCreateConductor

See warning list from a recent build:

```
RepositoryCreateConductor.cs(10,18): warning CS1591: Missing XML comment for publicly visible type or member 'RepositoryCreateConductor<T>' [C:\Users\bscot\acc\origin\AndcultureCode.CSharp.Conductors\src\AndcultureCode.CSharp.Conductors\AndcultureCode.CSharp.Conductors.csproj]
RepositoryCreateConductor.cs(15,21): warning CS1591: Missing XML comment for publicly visible type or member 'RepositoryCreateConductor<T>.CommandTimeout' [C:\Users\bscot\acc\origin\AndcultureCode.CSharp.Conductors\src\AndcultureCode.CSharp.Conductors\AndcultureCode.CSharp.Conductors.csproj]
RepositoryCreateConductor.cs(27,16): warning CS1591: Missing XML comment for publicly visible type or member 'RepositoryCreateConductor<T>.RepositoryCreateConductor(IRepository<T>)' [C:\Users\bscot\acc\origin\AndcultureCode.CSharp.Conductors\src\AndcultureCode.CSharp.Conductors\AndcultureCode.CSharp.Conductors.csproj]
RepositoryCreateConductor.cs(38,41): warning CS1591: Missing XML comment for publicly visible type or member 'RepositoryCreateConductor<T>.BulkCreate(IEnumerable<T>, long?)' [C:\Users\bscot\acc\origin\AndcultureCode.CSharp.Conductors\src\AndcultureCode.CSharp.Conductors\AndcultureCode.CSharp.Conductors.csproj]
RepositoryCreateConductor.cs(43,33): warning CS1591: Missing XML comment for publicly visible type or member 'RepositoryCreateConductor<T>.BulkCreateDistinct<TKey>(IEnumerable<T>, Func<T, TKey>, long?)' [C:\Users\bscot\acc\origin\AndcultureCode.CSharp.Conductors\src\AndcultureCode.CSharp.Conductors\AndcultureCode.CSharp.Conductors.csproj]
RepositoryCreateConductor.cs(48,35): warning CS1591: Missing XML comment for publicly visible type or member 'RepositoryCreateConductor<T>.Create(T, long?)' [C:\Users\bscot\acc\origin\AndcultureCode.CSharp.Conductors\src\AndcultureCode.CSharp.Conductors\AndcultureCode.CSharp.Conductors.csproj]
RepositoryCreateConductor.cs(53,41): warning CS1591: Missing XML comment for publicly visible type or member 'RepositoryCreateConductor<T>.Create(IEnumerable<T>, long?)' [C:\Users\bscot\acc\origin\AndcultureCode.CSharp.Conductors\src\AndcultureCode.CSharp.Conductors\AndcultureCode.CSharp.Conductors.csproj]
RepositoryCreateConductor.cs(58,33): warning CS1591: Missing XML comment for publicly visible type or member 'RepositoryCreateConductor<T>.CreateDistinct<TKey>(IEnumerable<T>, Func<T, TKey>, long?)' [C:\Users\bscot\acc\origin\AndcultureCode.CSharp.Conductors\src\AndcultureCode.CSharp.Conductors\AndcultureCode.CSharp.Conductors.csproj]

```

Add or fix comments as needed to reduce the number of warnings.

--- a/src/AndcultureCode.CSharp.Conductors/AndcultureCode.CSharp.Conductors.md
+++ b/src/AndcultureCode.CSharp.Conductors/AndcultureCode.CSharp.Conductors.md
@@ -3,6 +3,14 @@
 
 ## Contents
 
+- [RepositoryCreateConductor\`1](#T-AndcultureCode-CSharp-Conductors-RepositoryCreateConductor`1 'AndcultureCode.CSharp.Conductors.RepositoryCreateConductor`1')
+  - [#ctor(repository)](#M-AndcultureCode-CSharp-Conductors-RepositoryCreateConductor`1-#ctor-AndcultureCode-CSharp-Core-Interfaces-Data-IRepository{`0}- 'AndcultureCode.CSharp.Conductors.RepositoryCreateConductor`1.#ctor(AndcultureCode.CSharp.Core.Interfaces.Data.IRepository{`0})')
+  - [CommandTimeout](#P-AndcultureCode-CSharp-Conductors-RepositoryCreateConductor`1-CommandTimeout 'AndcultureCode.CSharp.Conductors.RepositoryCreateConductor`1.CommandTimeout')
+  - [BulkCreate(items,createdById)](#M-AndcultureCode-CSharp-Conductors-RepositoryCreateConductor`1-BulkCreate-System-Collections-Generic-IEnumerable{`0},System-Nullable{System-Int64}- 'AndcultureCode.CSharp.Conductors.RepositoryCreateConductor`1.BulkCreate(System.Collections.Generic.IEnumerable{`0},System.Nullable{System.Int64})')
+  - [BulkCreateDistinct\`\`1(items,property,createdById)](#M-AndcultureCode-CSharp-Conductors-RepositoryCreateConductor`1-BulkCreateDistinct``1-System-Collections-Generic-IEnumerable{`0},System-Func{`0,``0},System-Nullable{System-Int64}- 'AndcultureCode.CSharp.Conductors.RepositoryCreateConductor`1.BulkCreateDistinct``1(System.Collections.Generic.IEnumerable{`0},System.Func{`0,``0},System.Nullable{System.Int64})')
+  - [Create(item,createdById)](#M-AndcultureCode-CSharp-Conductors-RepositoryCreateConductor`1-Create-`0,System-Nullable{System-Int64}- 'AndcultureCode.CSharp.Conductors.RepositoryCreateConductor`1.Create(`0,System.Nullable{System.Int64})')
+  - [Create(items,createdById)](#M-AndcultureCode-CSharp-Conductors-RepositoryCreateConductor`1-Create-System-Collections-Generic-IEnumerable{`0},System-Nullable{System-Int64}- 'AndcultureCode.CSharp.Conductors.RepositoryCreateConductor`1.Create(System.Collections.Generic.IEnumerable{`0},System.Nullable{System.Int64})')
+  - [CreateDistinct\`\`1(items,property,createdById)](#M-AndcultureCode-CSharp-Conductors-RepositoryCreateConductor`1-CreateDistinct``1-System-Collections-Generic-IEnumerable{`0},System-Func{`0,``0},System-Nullable{System-Int64}- 'AndcultureCode.CSharp.Conductors.RepositoryCreateConductor`1.CreateDistinct``1(System.Collections.Generic.IEnumerable{`0},System.Func{`0,``0},System.Nullable{System.Int64})')
 - [RepositoryDeleteConductor\`1](#T-AndcultureCode-CSharp-Conductors-RepositoryDeleteConductor`1 'AndcultureCode.CSharp.Conductors.RepositoryDeleteConductor`1')
   - [#ctor(repository)](#M-AndcultureCode-CSharp-Conductors-RepositoryDeleteConductor`1-#ctor-AndcultureCode-CSharp-Core-Interfaces-Data-IRepository{`0}- 'AndcultureCode.CSharp.Conductors.RepositoryDeleteConductor`1.#ctor(AndcultureCode.CSharp.Core.Interfaces.Data.IRepository{`0})')
   - [CommandTimeout](#P-AndcultureCode-CSharp-Conductors-RepositoryDeleteConductor`1-CommandTimeout 'AndcultureCode.CSharp.Conductors.RepositoryDeleteConductor`1.CommandTimeout')
@@ -29,6 +37,141 @@
   - [BulkUpdate(items,updatedBy)](#M-AndcultureCode-CSharp-Conductors-RepositoryUpdateConductor`1-BulkUpdate-System-Collections-Generic-IEnumerable{`0},System-Nullable{System-Int64}- 'AndcultureCode.CSharp.Conductors.RepositoryUpdateConductor`1.BulkUpdate(System.Collections.Generic.IEnumerable{`0},System.Nullable{System.Int64})')
   - [Update(item,updatedBy)](#M-AndcultureCode-CSharp-Conductors-RepositoryUpdateConductor`1-Update-`0,System-Nullable{System-Int64}- 'AndcultureCode.CSharp.Conductors.RepositoryUpdateConductor`1.Update(`0,System.Nullable{System.Int64})')
   - [Update(items,updatedBy)](#M-AndcultureCode-CSharp-Conductors-RepositoryUpdateConductor`1-Update-System-Collections-Generic-IEnumerable{`0},System-Nullable{System-Int64}- 'AndcultureCode.CSharp.Conductors.RepositoryUpdateConductor`1.Update(System.Collections.Generic.IEnumerable{`0},System.Nullable{System.Int64})')
+
+<a name='T-AndcultureCode-CSharp-Conductors-RepositoryCreateConductor`1'></a>
+## RepositoryCreateConductor\`1 `type`
+
+##### Namespace
+
+AndcultureCode.CSharp.Conductors
+
+##### Summary
+
+Ability to create an entity or list of entities
+
+##### Generic Types
+
+| Name | Description |
+| ---- | ----------- |
+| T |  |
+
+<a name='M-AndcultureCode-CSharp-Conductors-RepositoryCreateConductor`1-#ctor-AndcultureCode-CSharp-Core-Interfaces-Data-IRepository{`0}-'></a>
+### #ctor(repository) `constructor`
+
+##### Summary
+
+Constructor
+
+##### Parameters
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| repository | [AndcultureCode.CSharp.Core.Interfaces.Data.IRepository{\`0}](#T-AndcultureCode-CSharp-Core-Interfaces-Data-IRepository{`0} 'AndcultureCode.CSharp.Core.Interfaces.Data.IRepository{`0}') |  |
+
+<a name='P-AndcultureCode-CSharp-Conductors-RepositoryCreateConductor`1-CommandTimeout'></a>
+### CommandTimeout `property`
+
+##### Summary
+
+Ability to set and get the underlying DbContext's command timeout
+
+<a name='M-AndcultureCode-CSharp-Conductors-RepositoryCreateConductor`1-BulkCreate-System-Collections-Generic-IEnumerable{`0},System-Nullable{System-Int64}-'></a>
+### BulkCreate(items,createdById) `method`
+
+##### Summary
+
+Ability to create entities using a list in a single bulk operation.
+
+##### Returns
+
+
+
+##### Parameters
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| items | [System.Collections.Generic.IEnumerable{\`0}](http://msdn.microsoft.com/query/dev14.query?appId=Dev14IDEF1&l=EN-US&k=k:System.Collections.Generic.IEnumerable 'System.Collections.Generic.IEnumerable{`0}') | List of items to create |
+| createdById | [System.Nullable{System.Int64}](http://msdn.microsoft.com/query/dev14.query?appId=Dev14IDEF1&l=EN-US&k=k:System.Nullable 'System.Nullable{System.Int64}') | Id of user creating the items |
+
+<a name='M-AndcultureCode-CSharp-Conductors-RepositoryCreateConductor`1-BulkCreateDistinct``1-System-Collections-Generic-IEnumerable{`0},System-Func{`0,``0},System-Nullable{System-Int64}-'></a>
+### BulkCreateDistinct\`\`1(items,property,createdById) `method`
+
+##### Summary
+
+Ability to create entities using a list in a single bulk operation without duplicates.
+
+##### Returns
+
+
+
+##### Parameters
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| items | [System.Collections.Generic.IEnumerable{\`0}](http://msdn.microsoft.com/query/dev14.query?appId=Dev14IDEF1&l=EN-US&k=k:System.Collections.Generic.IEnumerable 'System.Collections.Generic.IEnumerable{`0}') | List of items to create |
+| property | [System.Func{\`0,\`\`0}](http://msdn.microsoft.com/query/dev14.query?appId=Dev14IDEF1&l=EN-US&k=k:System.Func 'System.Func{`0,``0}') | Property used to remove duplicates |
+| createdById | [System.Nullable{System.Int64}](http://msdn.microsoft.com/query/dev14.query?appId=Dev14IDEF1&l=EN-US&k=k:System.Nullable 'System.Nullable{System.Int64}') | Id of user creating the items |
+
+<a name='M-AndcultureCode-CSharp-Conductors-RepositoryCreateConductor`1-Create-`0,System-Nullable{System-Int64}-'></a>
+### Create(item,createdById) `method`
+
+##### Summary
+
+Ability to create an entity
+
+##### Returns
+
+
+
+##### Parameters
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| item | [\`0](#T-`0 '`0') | Item to be created |
+| createdById | [System.Nullable{System.Int64}](http://msdn.microsoft.com/query/dev14.query?appId=Dev14IDEF1&l=EN-US&k=k:System.Nullable 'System.Nullable{System.Int64}') | Id of user creating the item |
+
+<a name='M-AndcultureCode-CSharp-Conductors-RepositoryCreateConductor`1-Create-System-Collections-Generic-IEnumerable{`0},System-Nullable{System-Int64}-'></a>
+### Create(items,createdById) `method`
+
+##### Summary
+
+Ability to create entities individually using a list
+
+##### Returns
+
+
+
+##### Parameters
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| items | [System.Collections.Generic.IEnumerable{\`0}](http://msdn.microsoft.com/query/dev14.query?appId=Dev14IDEF1&l=EN-US&k=k:System.Collections.Generic.IEnumerable 'System.Collections.Generic.IEnumerable{`0}') | List of items to be created |
+| createdById | [System.Nullable{System.Int64}](http://msdn.microsoft.com/query/dev14.query?appId=Dev14IDEF1&l=EN-US&k=k:System.Nullable 'System.Nullable{System.Int64}') | Id of user creating the items |
+
+<a name='M-AndcultureCode-CSharp-Conductors-RepositoryCreateConductor`1-CreateDistinct``1-System-Collections-Generic-IEnumerable{`0},System-Func{`0,``0},System-Nullable{System-Int64}-'></a>
+### CreateDistinct\`\`1(items,property,createdById) `method`
+
+##### Summary
+
+Ability to create entities individually without duplicates
+
+##### Returns
+
+
+
+##### Parameters
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| items | [System.Collections.Generic.IEnumerable{\`0}](http://msdn.microsoft.com/query/dev14.query?appId=Dev14IDEF1&l=EN-US&k=k:System.Collections.Generic.IEnumerable 'System.Collections.Generic.IEnumerable{`0}') | List of items to create |
+| property | [System.Func{\`0,\`\`0}](http://msdn.microsoft.com/query/dev14.query?appId=Dev14IDEF1&l=EN-US&k=k:System.Func 'System.Func{`0,``0}') | Property used to remove duplicates |
+| createdById | [System.Nullable{System.Int64}](http://msdn.microsoft.com/query/dev14.query?appId=Dev14IDEF1&l=EN-US&k=k:System.Nullable 'System.Nullable{System.Int64}') | Id of user creating the items |
+
+##### Generic Types
+
+| Name | Description |
+| ---- | ----------- |
+| TKey |  |
 
 <a name='T-AndcultureCode-CSharp-Conductors-RepositoryDeleteConductor`1'></a>
 ## RepositoryDeleteConductor\`1 `type`

--- a/src/AndcultureCode.CSharp.Conductors/RepositoryCreateConductor.cs
+++ b/src/AndcultureCode.CSharp.Conductors/RepositoryCreateConductor.cs
@@ -8,7 +8,7 @@ using AndcultureCode.CSharp.Core.Interfaces.Entity;
 namespace AndcultureCode.CSharp.Conductors
 {
     /// <summary>
-    /// Ability to create an entity or list of entities
+    /// Ability to create an entity or multiple entities
     /// </summary>
     /// <typeparam name="T"></typeparam>
     public class RepositoryCreateConductor<T> : Conductor, IRepositoryCreateConductor<T>

--- a/src/AndcultureCode.CSharp.Conductors/RepositoryCreateConductor.cs
+++ b/src/AndcultureCode.CSharp.Conductors/RepositoryCreateConductor.cs
@@ -7,11 +7,18 @@ using AndcultureCode.CSharp.Core.Interfaces.Entity;
 
 namespace AndcultureCode.CSharp.Conductors
 {
+    /// <summary>
+    /// Ability to create an entity or list of entities
+    /// </summary>
+    /// <typeparam name="T"></typeparam>
     public class RepositoryCreateConductor<T> : Conductor, IRepositoryCreateConductor<T>
         where T : class, IEntity
     {
         #region Properties
 
+        /// <summary>
+        /// Ability to set and get the underlying DbContext's command timeout
+        /// </summary>
         public int? CommandTimeout
         {
             get => _repository.CommandTimeout;
@@ -24,6 +31,10 @@ namespace AndcultureCode.CSharp.Conductors
 
         #region Constructor
 
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="repository"></param>
         public RepositoryCreateConductor(
             IRepository<T> repository
             )
@@ -35,26 +46,59 @@ namespace AndcultureCode.CSharp.Conductors
 
         #region IRepositoryCreateConductor
 
+        /// <summary>
+        /// Ability to create entities using a list in a single bulk operation.
+        /// </summary>
+        /// <param name="items">List of items to create</param>
+        /// <param name="createdById">Id of user creating the items</param>
+        /// <returns></returns>
         public virtual IResult<List<T>> BulkCreate(IEnumerable<T> items, long? createdById = default(long?))
         {
             return _repository.BulkCreate(items, createdById);
         }
 
+        /// <summary>
+        /// Ability to create entities using a list in a single bulk operation without duplicates.
+        /// </summary>
+        /// <param name="items">List of items to create</param>
+        /// <param name="property">Property used to remove duplicates</param>
+        /// <param name="createdById">Id of user creating the items</param>
+        /// <returns></returns>
         public IResult<List<T>> BulkCreateDistinct<TKey>(IEnumerable<T> items, Func<T, TKey> property, long? createdById = null)
         {
             return _repository.BulkCreateDistinct(items, property, createdById);
         }
 
+        /// <summary>
+        /// Ability to create an entity 
+        /// </summary>
+        /// <param name="item">Item to be created</param>
+        /// <param name="createdById">Id of user creating the item</param>
+        /// <returns></returns>
         public virtual IResult<T> Create(T item, long? createdById = default(long?))
         {
             return _repository.Create(item, createdById);
         }
 
+        /// <summary>
+        /// Ability to create entities individually using a list 
+        /// </summary>
+        /// <param name="items">List of items to be created</param>
+        /// <param name="createdById">Id of user creating the items</param>
+        /// <returns></returns>
         public virtual IResult<List<T>> Create(IEnumerable<T> items, long? createdById = default(long?))
         {
             return _repository.Create(items, createdById);
         }
 
+        /// <summary>
+        /// Ability to create entities individually without duplicates
+        /// </summary>
+        /// <typeparam name="TKey"></typeparam>
+        /// <param name="items">List of items to create</param>
+        /// <param name="property">Property used to remove duplicates</param>
+        /// <param name="createdById">Id of user creating the items</param>
+        /// <returns></returns>
         public IResult<List<T>> CreateDistinct<TKey>(IEnumerable<T> items, Func<T, TKey> property, long? createdById = null)
         {
             return _repository.CreateDistinct(items, property, createdById);

